wire-runtime-swift/src/test/swift/sample/Period.swift

```diff
@@ -1,6 +1,5 @@
 // Code generated by Wire protocol buffer compiler, do not edit.
 // Source: squareup.geology.Period in squareup/geology/period.proto
-import Foundation
```

Comment:
This is OK as our custom import system within Wire was always assuming Foundation is needed...